### ISSUE TITLE
Unable to attach attachment or image from attach icon using key board when screen reader is active.

### ISSUE THREAD

**Issue by prajwalmanju** [microsoft/AdaptiveCards#6784]
### Target Application

Microsoft Teams

### Application Operating System

Windows

### Schema Version

NA

### Problem Description

Unable to attach attachment or image from attach icon using key board when screen reader is active. After selecting any file and clicking on enter or selecting open button and clicking enter nothing is attaching in chat bot text box.

### Expected Outcome

when screen reader is enable Teams should able to take selected attachment via chat. 

### Actual Outcome

when screen reader is enabled Teams doesn't take attachment via chat.

### Card JSON

```JSON
NA
```

**Comment by Wajeed-msft** [comment]
This issue has been copied over to Microsoft Teams repo here: https://github.com/MicrosoftDocs/msteams-docs/issues/4731
